python/nammu/controller/NammuController.py

```diff
@@ -797,7 +797,7 @@ def editSettings(self, event=None):
         # Clear console in preparation for new settings messages
         self.consoleController.clearConsole()
 
-        edit_settings_controller = EditSettingsController(self)
```

Comment:
This comment fits better in this PR:

https://github.com/oracc/nammu/pull/390#discussion_r296770091

Reply:
I pushed a commit to [`giordano:cleanup`](https://github.com/giordano/nammu/tree/cleanup) a few minutes ago, but it's not showing up in the PR :neutral_face:  https://www.githubstatus.com/ says that GitHub is up and running, I have no idea what happened to the commit :confused: